internal/pkg/crawl/capture.go

```diff
@@ -275,6 +276,25 @@ func (c *Crawl) Capture(item *frontier.Item) error {
 				}
 			}
 		}
+	} else if facebook.IsFacebookPostURL(utils.URLToString(item.URL)) {
+		// Generate the embed URL
+		embedURL, err := facebook.GenerateEmbedURL(utils.URLToString(item.URL))
+		if err != nil {
+			c.Log.WithFields(c.genLogFields(err, item.URL, nil)).Error("error while generating Facebook embed URL")
+		} else {
+			if embedURL == nil {
+				c.Log.WithFields(c.genLogFields(err, item.URL, nil)).Error("error while generating Facebook embed URL")
+			} else {
+				// Create the embed item
+				embedItem := frontier.NewItem(embedURL, item, item.Type, item.Hop, item.ID, false)
+
+				// Capture the embed item
+				err = c.Capture(embedItem)
+				if err != nil {
```

Comment:
4 levels of indentation, nice! /s
We should really rewrite the site-specific part one day

Reply:
blablabla